modules/userId.js

```diff
@@ -414,6 +443,10 @@ export function init(config) {
       updateSubmodules();
     }
   })
+
+  // exposing getUserIds function in global-name-space so that userIds stored in Prebid can be used by external codes.
+  let theGlobal = getGlobal();
```

Comment:
@jaiminpanchal27 Can you review this use of getGlobal?

Reply:
@idettman this looks good. Use case here is valid and user id being a core module we can allow usage of global. 
Another approach to not use global here was to import getUserIds function in prebid.js and use their but then it will always import user id module.

Reply:
Thank you @jaiminpanchal27 👍